src/routes/routes.tsx

```diff
@@ -39,7 +45,11 @@ const router = createBrowserRouter([
 ]);
 
 const Routes: React.FC = () => {
-  return <RouterProvider router={router} />;
+  return (
+    <CertificatesProvider>
```

Comment:
Here you wrapped the hole app with the certificate context, but you don't need it in whole app but just on certificate view. Edit and Create views are comunicating with database (backend) so they don't need this context